datashader/tests/test_pandas.py

```diff
@@ -70,7 +72,7 @@ def test_sum():
 
 
 def test_min():
-    out = xr.DataArray(df.i64.reshape((2, 2, 5)).min(axis=2).astype('f8').T,
+    out = xr.DataArray(df.i64.values.reshape((2, 2, 5)).min(axis=2).astype('f8').T,
                        coords=coords, dims=dims)
```

Comment:
It removes the deprecation warning about `reshape`. It looks like it was deprecated in Pandas 0.19.0.